app/models/concerns/bank_holiday.rb

```diff
@@ -0,0 +1,18 @@
+module BankHoliday
+  include ActiveSupport::Concern
+
+  def is_bank_holiday?
+    read_bank_holidays.include?(self)
+  end
+
+  private
+
+  def read_bank_holidays
+    return [] unless File.exists?(bank_holidays_file)
+    Icalendar.parse(File.open(bank_holidays_file)).first.events.map(&:dtstart)
+  end
+
+  def bank_holidays_file
+    Rails.root.join("data", "bank_holidays.ics")
```

Comment:
Can we move the .ics file location to be in the Settings please?
Should be able to change location using an ENV variable, ok to default to /data/bank_holidays.ics otherwise